lib/Decl.cpp

```diff
@@ -289,7 +289,9 @@ llvm::json::Object ValueDecl::SerializeToJSON(const llvm::DataLayout &dl) {
 FunctionDecl FunctionDecl::Create(const llvm::Function &func,
                                   const llvm::Module &mdl) {
   const llvm::DataLayout& dl = mdl.getDataLayout();
-  const remill::Arch *arch = remill::Arch::GetModuleArch(mdl);
+
+  remill::Arch::ArchPtr arch = remill::Arch::GetModuleArch(mdl);
```

Comment:
Lets pass arch into this function, given that we're passing module into the function.

Reply:
Fixed e5d6087